docstore/driver/document.go

```diff
@@ -182,3 +182,15 @@ func (d Document) Decode(dec Decoder) error {
 	}
 	return decodeStruct(d.s, dec)
 }
+
+// RevisionOn returns whether or not we should perform optimistic locking for
```

Comment:
Specifically, this tells you whether you should write a revision value when you write a document. Let's make that clear.